spec/system/task_spec.rb

```diff
@@ -1,24 +1,25 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+
+  let(:project){create(:project)}
+  let(:task){create(:task)}
```

Comment:
冒頭の改行は詰めてしまって、波括弧（ブレース）はスペースを空けるのが見やすい書き方かと思いました。
```suggestion
  let(:project){ create(:project) }
  let(:task){ create(:task) }
```

Reply:
d1841bf 波括弧にスペースを加え修正しました。